controllers/usersController.js

```diff
@@ -3,14 +3,14 @@ const { getAllUsers, getUserById } = require('../queries/usersQueries');
 
 const usersController = express.Router();
 
-usersController.get('/', (req, res) => {
-  users = getAllUsers() || [];
+usersController.get('/', async (req, res) => {
+  users = await getAllUsers() || [];
```

Comment:
A small point but I think we don't need the or statement anymore. It can just be:
```
const users  = await getAllUsers()
```
This is because `getAllUsers()` will always return an array (because it uses `db.any`)
The original code also needed `const` or `let` here